std/algorithm/comparison.d

```diff
@@ -61,7 +61,7 @@ import std.range.primitives;
 import std.traits;
 // FIXME
 import std.typecons; // : tuple, Tuple, Flag;
-import std.meta : allSatisfy;
+import std.meta; // : allSatisfy, anySatisfy;
```

Comment:
This the common style in Phobos as I think the idea is that we can use a simple regex to flip the switch once the deprecation period for the exposed import is over.